code/game/gamemodes/game_mode.dm

```diff
@@ -272,8 +272,9 @@ Implants;
 	return 0
 
 
-/datum/game_mode/proc/send_intercept()
-	var/intercepttext = "<FONT size = 3><B>Cent. Com. Update</B> Requested status information:</FONT><HR>"
+/datum/game_mode/proc/send_intercept() //Disabled.
+	return
+/*	var/intercepttext = "<FONT size = 3><B>Cent. Com. Update</B> Requested status information:</FONT><HR>"
 	intercepttext += "<B> In case you have misplaced your copy, attached is a list of personnel whom reliable sources&trade; suspect may be affiliated with the Syndicate:</B><br>"

```

Comment:
because rng "oh now you have extra TCs" can fuck with the balance of costs on some shit

some things are balanced around "you can only buy 1"